phy/cluster/manual/session.py

```diff
@@ -689,7 +692,7 @@ def _create_view_model(self, name, **kwargs):
         vm_class = _VIEW_MODELS[name]
         return vm_class(self.model, store=self.cluster_store, **kwargs)
 
-    def show_waveforms(self):
+    def show_waveforms(self, show=True):
```

Comment:
ok so we change that to:

* `create_waveform_view()` returns a VisPy canvas
* `show_waveforms()` stays as it is (but it leverages `create_waveform_view()` under the hood)